packages/design-system-docs/src/scripts/components/ReactPropDocs.jsx

```diff
@@ -19,35 +27,22 @@ class ReactPropDocs extends React.PureComponent {
   }
 
   render() {
-    // Specify ARIA roles attribute for table to ensure responsive HTML table is accessible to screen readers
     return [
       <h3 key="propDocsHeader">Props</h3>,
-      <div key="propDocsTable" className="docs_table--container">
-        <table className="ds-c-table ds-c-table--borderless docs_table" role="table">
-          <caption>
-            <span className="ds-u-padding--1 ds-u-visibility--screen-reader">
-              React Properties Documentation
-            </span>
-          </caption>
-          <thead>
-            <tr role="row">
-              <th id="columnname" role="columnheader" scope="col">
-                Name
-              </th>
-              <th id="columntype" role="columnheader" scope="col">
-                Type
-              </th>
-              <th id="columndefault" role="columnheader" scope="col">
-                Default
-              </th>
-              <th id="columndescription" role="columnheader" scope="col">
-                Description
-              </th>
-            </tr>
-          </thead>
-          <tbody>{this.rows()}</tbody>
-        </table>
-      </div>,
+      <Table key="propDocsTable" stacked="sm" scrollTable>
+        <TableCaption className="ds-u-padding--1 ds-u-visibility--screen-reader">
+          React Properties Documentation
```

Comment:
I don't think a caption here is necessary considering we have the `h3` saying props.

Reply:
I'd like to see us using the `caption` instead of the `h3` as it keeps the component tighter together and more relevant to the content. An `h3` could lose the hierarchy on the page or need to be changed while a caption is directly tied to the table. 

I'd be curious as to what @davidakennedy thinks here

Reply:
If we set `scrollTable` true, this is needed to tie back to `tabIndex`

Reply:
Good thoughts! Tables don't always have to have a caption. So it's okay if it doesn't have one, and you relied on the h3. It all depends on the content around the table, and how complex the table is in terms of content.

To get the best of both worlds, you could put a h3 inside the caption. That way, the heading structure is there and you're not repeating info. That would be a good way to go here.

Reply:
I'm good with an `h3` inside the caption as long as the styles don't change. And @nichia, what do you mean its needed to tie back to `tabIndex`? I thought `tabIndex` is for making the table focusable, and we decided in this iteration not to worry about adding additional focus functionality